src/main/java/org/apache/maven/shared/dependency/analyzer/DefaultProjectDependencyAnalyzer.java

```diff
@@ -25,17 +25,11 @@
 import java.io.File;
 import java.io.IOException;
 import java.net.URL;
+import java.util.*;
```

Comment:
please don't use asterix in imports

Reply:
whoops, we don't use asterix either, so this was an ide glitch on my end. will update.